frontend/model/state.js

```diff
@@ -28,7 +28,8 @@ const initialState = {
   namespaceLookups: Object.create(null), // { [username]: sbp('namespace/lookup') }
   periodicNotificationAlreadyFiredMap: {}, // { notificationKey: boolean },
   contractSiginingKeys: Object.create(null),
-  lastLoggedIn: {} // Group last logged in information
+  lastLoggedIn: {}, // Group last logged in information
+  preferences: {}
```

Comment:
Any time we modify the state like this we need to add an entry in `postUpgradeVerification`